homeassistant/bootstrap.py

```diff
@@ -42,6 +44,10 @@
     "mqtt_eventstream",
     # To provide account link implementations
     "cloud",
+    # Get the frontend up and running as soon
+    # as possible so problem integrations can
+    # be removed
+    "frontend",
```

Comment:
We also need `config` or else we don't have APIs to manage the config entries.

```suggestion
    "frontend",
    "config",
```